app/locales/en.json

```diff
@@ -1,5 +1,30 @@
 {
   "app.name": "Cobuy",
   "agents.nameLabel": "name",
-  "agents.descriptionLabel": "description"
+  "agents.descriptionLabel": "description",
+  "agents.logout": "log out",
+  "agents.email": "email",
+  "agents.role": "role",
+  "agents.password": "password",
+  "agents.confirmPassword": "confirm password",
+  "agents.groupName": "group name",
+  "agents.addMember": "add member",
+  "agents.removeMember": "remove member",
+  "agents.editProfile": "edit profile",
+  "agents.saveProfile": "save profile",
+  "agents.createAccount": "create new account",
+  "agents.signIn": "sign in",
+  "agents.welcome": "hey, welcome to cobuy!",
+  "agents.signInWith": "sign in with...",
+  "agents.saveAvatar": "save avatar",
+  "agents.editAvatar": "edit avatar",
+  "supply.removeOffering": "remove offering",
+  "supply.addOffering": "add offering",
+  "supply.quantity": "quantity",
+  "supply.unit": "unit",
+  "supply.litres": "litres",
```

Comment:
is `en` American English or British English? maybe also an opportunity to add `en-US` and `en-GB` locales, for whichever is not our default (which we should document somewhere, otherwise i'll write American English as my default).